database_admin/schema/create_schema.sql

```diff
@@ -785,6 +784,8 @@ CREATE INDEX IF NOT EXISTS
 
 GRANT SELECT, INSERT, UPDATE, DELETE ON advisory_metadata TO evaluator;
 GRANT SELECT, INSERT, UPDATE, DELETE ON advisory_metadata TO vmaas_sync;
+-- TODO: Remove
+GRANT SELECT, INSERT, UPDATE, DELETE ON advisory_metadata TO listener;
```

Comment:
see above; the two line has been removed in master

Reply:
I know, it will be fixed in a moment. Sorry you got a ping tho :neutral_face: I just had to merge master, because we had conflicts.